converter/src/main/java/gov/cms/qpp/conversion/model/validation/MeasureConfig.java

```diff
@@ -31,6 +31,7 @@
 
 	@JsonProperty("eMeasureUuid")
 	private String electronicMeasureVerUuid;
+	private String cpcPlusGroup;
```

Comment:
does this need a @JsonProperty? Do we even need this annotation? I'm a little unfamiliar with the json pojo framework we're using.

Reply:
Usually you use `@JsonProperty` when you want the JSON emitter to output a different field name than the actual field name in the object you're serializing.  Looks like that's what's going on here, but @clydet can verify.

Reply:
Gotcha. My knowledge of Jackson's Json engine is limited, thanks for clearing that up.

Reply:
👍 @sfradkin